letsencrypt/models/letsencrypt.py

```diff
@@ -8,26 +8,26 @@
 import subprocess
 import tempfile
 from openerp import _, api, models, exceptions
-from openerp.tools.config import _get_default_datadir
+from openerp.tools import config
 
 
 DEFAULT_KEY_LENGTH = 4096
 _logger = logging.getLogger(__name__)
 
 
+def get_data_dir():
```

Comment:
Well, inheritance is a good point.
I change this because, in multi-database use case, we haven't request.env defined, so we can't call to models.
Do you know another way to do it?